bundles/org.palladiosimulator.analyzer.slingshot.eventdriver/src/org/palladiosimulator/analyzer/slingshot/eventdriver/internal/BusImplementation.java

```diff
@@ -153,22 +181,25 @@ private void searchForSubscribers(final CompositeDisposable composite,
 		}
 
 
-		System.out.println("\tAdded subscriber method " + method.getName());
-		composite.add(
-				this.bus.ofType(eventClass)
-						.doOnNext(ev -> System.out.println("ON NEXT " + ev.getClass().getSimpleName()))
-						.subscribe(
-								new AnnotatedSubscriber(method, object, compositeInterceptor, compositeInterceptor, subscribeAnnotation),
-								error -> {
-									System.out.println("Error happened: " + error.getClass().getSimpleName() + ":: " + error.getMessage());
-							    	this.exceptionHandlers.keySet().stream()
-							    		.filter(exClazz -> exClazz.isAssignableFrom(error.getClass()))
-							    		.flatMap(exClazz -> this.exceptionHandlers.get(exClazz).stream())
-							    		.forEach(exHandler -> exHandler.accept(error));
-							    }
-						)
-		);
+		this.register(AnnotatedSubscriber.fromJavaMethod(eventClass, object, method, subscribeAnnotation, compositeInterceptor, compositeInterceptor).build());
+	}
+
+	private void doError(final Throwable error) {
+		LOGGER.error("An error occurred: " + error.getClass().getSimpleName() + ":: " + error.getMessage());
```

Comment:
actually, i prefer to log the entire stacktrace. makes debugging a lot easier, imho :x
(nothing to block a merge though.)

Reply:
I think simply putting the `error` parameter as a second argument to the `LOGGER.error` method should make this possible